calculator.js

```diff
@@ -1,2 +1,11 @@
-var calculator = {
-}
+var x = 2;
+var y = 4;
+
+var add = x + y;
+var sub = x - y;
+var mult = x * y;
+var dvd = x / y;
+var pow = Math.pow(x, y);
+var sqrt = Math.sqrt(y);
+
```

Comment:
I like that you worked out an alternate approach to the problem even though you were lost. Great instinct. Let's meet up and go over this, I bet we can knock it out in a few minutes!